docker/docker-compose.state-sync.yml

```diff
@@ -3,7 +3,7 @@ services:
 
   lava-node-init:
     # Update with the most recent supported version of lavad
-    image: ghcr.io/lavanet/lava/lavad:${LAVAD_VERSION:-v2.0.0}
+    image: ghcr.io/lavanet/lava/lavad:${LAVAD_VERSION:-main}
```

Comment:
`main` is by definition unstable, I would use `latest` or the latest stable version `v2.2.0`

Reply:
Done